bin/spectatord_main.cc

```diff
@@ -54,6 +54,8 @@ ABSL_FLAG(bool, enable_statsd, false, "Enable statsd support.");
 ABSL_FLAG(bool, enable_unix_socket, true, "Enable UNIX domain socket support.");
 ABSL_FLAG(absl::Duration, meter_ttl, absl::Minutes(15),
           "Meter TTL: expire meters after this period of inactivity.");
+ABSL_FLAG(int, age_gauge_limit, 1000,
+          "The maximum number of age gauges that may be reported by this process.");
```

Comment:
Since gauges with a long TTL have the same problem, should the limit include all gauges?

Reply:
The risk is similar, but not exactly comparable, because long TTL gauges are eventually cleaned up, while age gauges are not. We are planning to provide a mechanism for deleting age gauges, but that will be submitted as a separate PR.